database/src/main/kotlin/no/nav/su/se/bakover/database/utbetaling/UtbetalingPostgresRepo.kt

```diff
@@ -97,6 +97,7 @@ internal class UtbetalingPostgresRepo(
             "utbetalingId" to utbetalingId,
             "forrigeUtbetalingslinjeId" to utbetalingslinje.forrigeUtbetalingslinjeId,
             "belop" to utbetalingslinje.beløp,
+            "uforegrad" to utbetalingslinje.uføregrad?.value
```

Comment:
```suggestion
            "uforegrad" to utbetalingslinje.uføregrad!!.value
```

Vi bør ikke støtte og kunne inserte null, bare lese ut null.

Reply:
Vi lagrer vel utbetalinger som stoppes / gjennopptas, disse kan være at dem ikke har uføregrad fra før. Fra hvordan jeg forstår hvordan dem funksjonene funker. Da vil vi få exception inn mot DB

Reply:
@jacob-meidell Kan det tenkes at vi kan fylle inn uføregraden i start/stopp-meldingene fra vedtaket/behandlinga/vilkåret for de som mangler det?

Reply:
Det er nok mulig for oss å få til noe, men se min tidligere kommentar ang praktisk gjennomførbarhet mot OS.

Reply:
med tanke på at vi fortsatt lagrer utbetalinger når dem stanses / gjenopptas, og eksisterende utbetalinger ikke har uføregrad, så må den være nullable